tailwind.config.ts

```diff
@@ -16,5 +16,5 @@ export default withTV({
       },
     },
   },
-  plugins: [themingSystemPlugin],
+  plugins: [themingSystemPlugin, require("tailwindcss-animate")],
```

Comment:
Can we import it as a ts module instead of doing `require`?

Maybe we can just do 
`import tailwindCssAnimatePlugin from 'tailwindcss-nimate'`

Then use `tailwindCssAnimatePlugin` here

Reply:
sure! added it at 8b42bdb